### ISSUE TITLE
MDDialog not working prperly on Android

### ISSUE THREAD

**Issue by fnadalt** [kivymd/KivyMD#800]
### Description of the Bug

MDDialog is not working properly on Android. Clickcs/Touches outside certain limited area in the MDDialog causes it to be dismissed/closed. Cannot be used on Android. I replicated the bug using the example code in the official KivyMD documentation (https://kivymd.readthedocs.io/en/latest/components/dialog/).

### Code and Logs

```python
from kivy.lang import Builder

from kivymd.app import MDApp
from kivymd.uix.button import MDFlatButton
from kivymd.uix.dialog import MDDialog

KV = '''
FloatLayout:

    MDFlatButton:
        text: "ALERT DIALOG"
        pos_hint: {'center_x': .5, 'center_y': .5}
        on_release: app.show_alert_dialog()
'''


class Example(MDApp):
    dialog = None

    def build(self):
        return Builder.load_string(KV)

    def show_alert_dialog(self):
        if not self.dialog:
            self.dialog = MDDialog(
                text="Discard draft?",
                buttons=[
                    MDFlatButton(
                        text="CANCEL", text_color=self.theme_cls.primary_color
                    ),
                    MDFlatButton(
                        text="DISCARD", text_color=self.theme_cls.primary_color
                    ),
                ],
            )
        self.dialog.open()


Example().run()
```

### Screenshots

Clicks outside the green rectangles causes the dialog to be closed/dismissed... In some cases the green area is smaller or even seems to be absent.
![Captura de pantalla de 2021-01-29 13-10-05](https://user-images.githubusercontent.com/12092450/106315925-b5ac4500-624a-11eb-859b-73a374e321af.png)

### Versions

* OS: Linux 5.10.7-arch1-1 x86_64
* Python: 3.9.1
* Kivy: 2.0.0
* KivyMD: 0.104.1

**Comment by HeaTTheatR** [comment]
@fnadalt We do not support old versions of the KivyMD library. This means that the new APIs listed in the documentation are only compatible with the master branch of the library. Try using the KivyMD library from the master branch.

**Comment by mp-007** [comment]
@fnadalt  See https://github.com/HeaTTheatR/KivyMD/issues/321
if you want to fix it in kivymd 0.104.1. Otherwise use KivyMD library from the master branch.
